pkgs/top-level/all-packages.nix

```diff
@@ -31057,4 +31057,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  xplr = callPackage ../applications/misc/xplr {};
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
I think it belongs in the same category as `ranger`, `nnn`, `vifm` etc. i.e. `applications/misc`. Sorted.